nomad/local/local_grapl_registry.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -euo pipefail
+
+GRAPL_LOCAL_REGISTRY_PORT=5000
+GRAPL_LOCAL_REGISTRY_NAME="grapl_local_registry"
+
+if [[ ! -z "$(docker ps --quiet --filter name="${GRAPL_LOCAL_REGISTRY_NAME}")" ]]; then
+    echo "already running."
+elif [[ ! -z "$(docker container ls --all --quiet --filter name="${GRAPL_LOCAL_REGISTRY_NAME}")" ]]; then
+    echo "Container exists but stopped. Restarting.."
+    docker start "${GRAPL_LOCAL_REGISTRY_NAME}"
```

Comment:
How does the container get stopped?

Would it be useful to add the `--rm` flag in the `docker run` command below?

Reply:
Gotcha; we could do that.